integration_test/third_party_apps_data/agent/ops-agent/linux/enable_elasticsearch

```diff
@@ -16,6 +16,16 @@ sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+metrics:
+  receivers:
+    elasticsearch_metrics:
+      type: elasticsearch
+      collection_interval: 30s
```

Comment:
Remove `collection_interval` as it's no longer a required param?

Reply:
For these tests, I noticed that some (all?) of them lower the collection interval down to 30s for kokoro. 

I'll remove it and change the sleep 30 to sleep 60, since it seems like we prefer that (better to test default config, I suppose).

Reply:
Yup! Thanks for addressing this.